config/generateKonfig.coffee

```diff
@@ -127,6 +127,7 @@ module.exports = (options, credentials) ->
 
     terraformerSecretKey: credentials.terraformer.secretKey
 
+  credentials.countly.host = 'http://dev.koding.com:32768/countly' if options.countlyPath
```

Comment:
this is done [here](https://github.com/koding/vault/pull/68) we can remove this line